src/api/roles/client/client.py

```
@@ -89,4 +89,42 @@ def me(db = Depends(get_session), acc: Account = Depends(get_client_account)):
     return ClientAccountResponse(
         base_account=acc,
         client_account=db.get(Client, acc.client_id)
-    )
\ No newline at end of file
+    )
+
+
+@router.post("/upload_progress_picture")
+def upload_progress_picture(file: UploadFile, acc: Account = Depends(get_client_account)):
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)